rgd/settings.py

```diff
@@ -13,6 +13,26 @@
 from configurations import values
 
 
+class GdalMinioMixin(ConfigMixin):
+    """Setup S3-like environment variables for Minio setups."""
+
+    @staticmethod
+    def before_binding(configuration: Type[ComposedConfiguration]):
+        # `MINIO_STORAGE_ACCESS_KEY` and `MINIO_STORAGE_SECRET_KEY`
+        # can only be accessed via environment variables
+        os.environ.setdefault(
+            'AWS_ACCESS_KEY_ID',
+            os.environ['DJANGO_MINIO_STORAGE_ACCESS_KEY'],
```

Comment:
To be stable with `composed-configuration`, these should be accessed via settings, not directly via the environment.

Reply:
Both `configuration.MINIO_STORAGE_ACCESS_KEY` and `configuration.MINIO_STORAGE_SECRET_KEY` seems to be a `configurations.values.SecretValue` with a value of `None`. Any idea how to work around this?